devlib/platform/gem5.py

```diff
@@ -127,6 +129,11 @@ def _start_interaction_gem5(self):
             # telnet connection.
             self.stderr_filename = f
 
+            # Enable power modelling if required
+            if self.power_models_dir:
```

Comment:
What other activation methods/scripts currently exist? Is there a reason there cannot be a standard interface for these?

Maybe we can still have some lowest-common-denominator configuration, e.g.  the model could take `power_model_args` and `power_model_env_vars` parameters.